src/main/java/fi/aalto/cs/intellij/notifications/APlusNotifications.java

```diff
@@ -0,0 +1,40 @@
+package fi.aalto.cs.intellij.notifications;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import fi.aalto.cs.intellij.common.BuildInfo;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A container class for notifications used by A+ plugin.
+ */
+public class APlusNotifications {
```

Comment:
I see quite little gain from having a container for "A+" notifications, but let's discuss it!

Reply:
if we aim at creating some generic base for A+ notification, then I'd propose something similar to this:
```java
public class APlusNotification extends Notification {

  public APlusNotification(@Nls(capitalization = Capitalization.Sentence) @NotNull String title,
      @NotNull String content,
      @NotNull NotificationType type) {
    // so to emphasize, that this is an "A+" notification
    // eventually, we might need more constructors to match "Notification"s own
    super("A+", title, content, type);
  }
}
```

but, again, the gain from this is quite modest, as there might be a need for more constructors to match the IJ's API.

Reply:
a drawback, that the https://github.com/Aalto-LeTech/intellij-plugin/pull/51/files#diff-4285c89d754ceac24452c9ab2f58cde1 will keep growing...

Reply:
I agree it might be unnecessary. Why I ended up creating a class for this (only once used) notification, was that I wanted to write separate unit tests to notifications content and whether it is dispatched or not. So I tested the `notification.getMessage()`, `notification.getTitle()` etc. in *APlusNotificationsTest.java* and in *VersionCheckerActivityTest.java* I only tested that the notification is an instance of the correct class and that its state (indicated by `getVersion()`) is correct.